controllers/news.controllers.js

```diff
@@ -9,13 +9,17 @@ exports.getTopics = (req, res) => {
 
 exports.getArticlesById = (req, res, next) => {
     const articleId = req.params.article_id
-    fetchArticlesById(articleId)
-        .then((article) => {
-            res.status(200).send({article})
+    Promise.all([fetchArticlesById(articleId),countCommentsByArticle(articleId)])
+        .then(([article, comment_count]) => {
+            const retrunObject = {article, comment_count}
```

Comment:
This is not fine, comment_count should be a key/value inside the object and you should still send a single {article}